build-system/tasks/integration.js

```diff
@@ -37,7 +37,11 @@ class Runner extends RuntimeTestRunner {
       return;
     }
     execOrDie('gulp clean');
-    execOrDie(`gulp dist --fortesting --config ${argv.config}`);
+    if (argv.compiled) {
+      execOrDie(`gulp dist --fortesting --config ${argv.config}`);
+    } else {
+      execOrDie(`gulp build --config ${argv.config}`);
+    }
```

Comment:
@powerivq `gulp integration` defaults to running against compiled code when `--nobuild` is not passed in. This PR should have changed this block:

https://github.com/ampproject/amphtml/blob/db49f436b562d99df0d005bf12f8f61c11c4af80/build-system/tasks/runtime-test/helpers.js#L161-L165

There might be other places that need to be changed as well.

Reply:
Thanks. Defaulting to testing compiled code is great. I am seeing it actually using unminimized version of files though. I can talk a further look at why.